### DISCUSSION TITLE
How to embed the developer portal in a corporate site?

### DISCUSSION THREAD

**Discussion by diegocantelli** [Azure/api-management-developer-portal#2632] [General]
Hi!

I would like to be able to embed it into an existing corporate site that is already describing our company, our products, even some APIs documentation in order to have the possibility to run the APIs from there. In this way, we will decrease the number of websites a customer must use in order to access all of our features. It can be annoying to have a website for the company and then another website just for running APIs and then maybe another website for something else, etc.

Is it possible? I couldn't find in the documentation available.

Thanks.

**Comment by mikebudzynski** [comment]
@diegocantelli Currently the managed developer portal doesn't allow for injection into an iframe - we'll see if we can change it. In the meantime, you can use the self-hosted version and inject it into an iframe in your corporate site.

**Comment by diegocantelli** [comment]
@mikebudzynski Thanks for the reply!

**Comment by azaslonov** [comment]
Update: we started working on the solution for this issue.

**Comment by eluchsinger** [comment]
@azaslonov any news here?
